src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -13,12 +14,42 @@
 #include "ray/common/ray_object.h"
 #include "ray/core_worker/context.h"
 #include "ray/core_worker/store_provider/memory_store_provider.h"
+#include "ray/core_worker/transport/dependency_resolver.h"
 #include "ray/gcs/redis_gcs_client.h"
 #include "ray/rpc/grpc_server.h"
 #include "ray/rpc/worker/core_worker_client.h"
 
+namespace {
+
+/// Extend std::priority_queue to allow std::move out of the priority queue.
+template <class T, class Container = std::vector<T>,
```

Comment:
Have you considered absl::btree_map<int64, T> or std::map<int64, T>? (The latter is already used on the receiver side as a ordered queue).

That should avoid needing to define this, or any comparators.

Reply:
Ah thanks, I didn't think of that. I'll switch it to one of these.